pre/remap_restart/remap_questions.py

```diff
@@ -124,7 +128,11 @@ def ask_questions():
         {
             "type": "text",
             "name": "output:shared:agrid",
-            "message": "Enter new atmospheric grid: \n C12   C180  C1000  C270\n C24   C360  C1440  C540\n C48   C500  C2880  C1080\n C90   C720  C5760  C2160  C1536\n ",
+            "message": "Enter new atmospheric grid: \n \
```

Comment:
Just a question, but can `message` accept a triple-quoted string? Then maybe we wouldn't need the `\n`?

Reply:
Yes, it accepts. That is python's feature.